modules/install-and-configure-oadp-kubevirt.adoc

```diff
@@ -8,7 +8,7 @@
 
 As a cluster administrator, you install {oadp-short} by installing the {oadp-short} Operator.
 
-The Operator installs link:https://velero.io/docs/v{velero-version}[Velero {velero-version}].
+The latest {oadp-short} Operator installs link:https://velero.io/docs/v{velero-version}[Velero {velero-version}].
```

Comment:
@ousleyp @weshayutin @amastbau 

Dalia Frank suggested - `this part do not refernce to a specific oadp version, so how do specify the velero that supposed to be different between oadp versions (as far to my knowledge)`

If I add it is the latest OADP operator, are you happy?

Reply:
I might personally do "The latest version of the {oadp-short} Operator," but otherwise it sounds good to me.

Reply:
Changed to 

```
The latest version of the {oadp-short} Operator installs link:https://velero.io/docs/v{velero-version}[Velero {velero-version}].
```

Reply:
@amastbau && @weshayutin - are you ok with this change, please?

Then I can annoy @ousleyp about finally getting it merged

Thanks